facebook_scraper/extractors.py

```diff
@@ -203,6 +203,12 @@ def extract_time(self) -> PartialPost:
             except (KeyError, ValueError):
                 continue
 
+        date = utils.parse_datetime(element_full_text=self.element.full_text)
```

Comment:
How complicate would it be to select just the html element with the date?

Reply:
I do not understand your suggestion

Reply:
I mean using `self.element.find('abbr', first=True).text` instead of `self.element.full_text`, since the date seems to always be inside an `abbr` tag.

Reply:
I'll add that change and merge it.